app/com/gu/contentapi/sanity/support/HttpRequestSupport.scala

```diff
@@ -1,16 +1,35 @@
 package com.gu.contentapi.sanity.support
 
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
 import com.gu.contentapi.sanity.Config
+import com.typesafe.config.ConfigFactory
+import java.io.File
 import org.scalatest.exceptions.TestPendingException
 import org.scalatest.{Assertions, Matchers}
 import org.scalatest.concurrent.ScalaFutures
-import play.api.libs.ws.{WSResponse, WSAuthScheme, WS, WSRequest}
+import play.api._
+import play.api.libs.ws._
+import play.api.libs.ws.ahc._
 import play.api.Play.current
 import scala.concurrent.duration._
 
 trait HttpRequestSupport extends ScalaFutures with Matchers with Assertions {
+  import scala.concurrent.ExecutionContext.Implicits._
 
-  def request(uri: String): WSRequest = WS.url(uri).withRequestTimeout(10000.millis)
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+
+  val configuration = Configuration.reference ++ Configuration(ConfigFactory.parseString(
+  """
+    |ws.followRedirects = true
+  """.stripMargin))
+
+  val environment = Environment(new File("."), this.getClass.getClassLoader, Mode.Prod)
+  val wsConfig = AhcWSClientConfigFactory.forConfig(configuration.underlying, environment.classLoader)
+  val wsClient: WSClient = AhcWSClient(wsConfig)
```

Comment:
Need to `.close` this when work is done.

Reply:
Re-read the code and I understand the tests run forever, so the client should never actually be closed.